.githooks/post-commit

```diff
@@ -5,6 +5,12 @@ if ! hash astyle; then
 	exit 1
 fi
 
+cargo fmt --version >/dev/null 2>&1
+if [[ $? != 0 ]]; then
+	echo "Warning: \`cargo fmt\` is not available."
```

Comment:
I think the warning messages for missing `cargo fmt` or `astyle` should be written similarly. For the same class of warnings, we should write the warning messages in the same form.